templates/README

```diff
@@ -1 +1,60 @@
-This contains some basic description about the BIDS dataset.
+### Introduction
+The README is usually the starting point for researchers using your data
+and serves as a guidepost for users of your data. A clear and informative
+README makes your data much more usable.
+
+Overview
+
+A paragraph giving an overview of the experiment. This should include the 
+goals or purpose and a discussion about how the experiment tries to achieve
+these goals.
+
+A brief overview of the tasks in the experiment.
+
+A brief discussion of condition variables (sometimes called contrasts
+or independent variables) that were varied across the experiment.
+
+A brief discussion of the response variables (sometimes called the
+dependent variables) that were measured and or calculated to assess
+the effects of varying the condition variables. This might also include
+questionnaires administered to assess behavioral aspects of the experiment.
+
+A brief discussion of the control variables --- that is what aspects 
+were explicitly controlled in this experiment. The control variables might 
+include subject pool, environmental conditions, set up, or other things 
+that were explicitly controlled.
+
+More details on all of this should be included in the Methods section.
+
+
+Methods  
+
+Subjects: A brief sentence about the subject pool in this experiment.
```

Comment:
```suggestion
Subjects: A brief sentence about the subject pool in this experiment (e.g. control).
```

Brief note: Control/DX status could be defined in the `participants.tsv` using a `group` column. Though, I think having this specified in the README can be helpful too.